src/main/java/com/example/board/controller/UserController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.board.controller;
+
+import com.example.board.domain.auth.JWTTokenProvider;
+import com.example.board.request.SignInRequest;
+import com.example.board.request.SignUpRequest;
+import com.example.board.response.SignInResponse;
+import com.example.board.service.UserService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/user")
+@RestController
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping("/signup")
+    public Long signUp(@RequestBody SignUpRequest signUpRequest) {
+        return userService.signUp(signUpRequest);
+    }
+
+    @PostMapping("/signin")
+    public ResponseEntity<HttpStatus> signIn(@RequestBody SignInRequest signInRequest) {
+        SignInResponse response = userService.signIn(signInRequest);
+        HttpHeaders httpHeaders = new HttpHeaders();
+        httpHeaders.add(JWTTokenProvider.HEADER_STRING, response.getToken());
+        return ResponseEntity.ok().headers(httpHeaders).body(HttpStatus.ACCEPTED);
```

Comment:
```suggestion
        String token = userService.signIn(signInRequest);
        HttpHeaders httpHeaders = new HttpHeaders();
        httpHeaders.add(JWTTokenProvider.HEADER_STRING, token);
        return ResponseEntity.ok().headers(httpHeaders);
```
- 지역변수는 의미있는 값으로 표기해주는 것이 가독성에 좋습니다.
- ACCEPTED (202)는 제가 알기로 "처리 요청이 수락되었지만, 완료되지 않았다"는 의미로 알고 있어요
- ResponseEntity.ok() 메소드를 사용하면 status로 200을 담아서 보냅니다.